### PR TITLE
Add api for nova service list

### PR DESCRIPTION
**Pull request by esommar**
Api implementation for nova service-list. Test included.

### CODE REVIEW

File: core/src/main/java/org/openstack4j/api/compute/ext/ServicesService.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import java.util.List;
+
+import org.openstack4j.model.compute.ext.Service;
+
+/**
+ * API which supports the "os-services" extension.
+ *
+ * @author Stephan Latour
+ */
+public interface ServicesService{
```
Comment (gondor) [inline core/src/main/java/org/openstack4j/api/compute/ext/ServicesService.java]: Could you prefix a space before the brace

**Comment by gondor** [comment]
Syntax comment, otherwise LGTM

File: core/src/main/java/org/openstack4j/api/compute/ext/ServicesService.java (hunk 1 above)
Comment (esommar) [inline core/src/main/java/org/openstack4j/api/compute/ext/ServicesService.java]: Thanks, done!

**Comment by auhlig** [comment]
Cool. LGTM. Many thanks @esommar and @gondor :)

**Pull request merged**